substation-diagram/substation-diagram-core/src/main/java/com/powsybl/substationdiagram/svg/DefaultSubstationDiagramStyleProvider.java

```diff
@@ -37,48 +38,44 @@
     @Override
     public Optional<String> getNodeStyle(Node node, boolean avoidSVGComponentsDuplication) {
         Objects.requireNonNull(node);
-        if (node.getType() == Node.NodeType.SWITCH && !avoidSVGComponentsDuplication) {
-            try {
-                StringBuilder style = new StringBuilder();
-                String className = escapeClassName(URLEncoder.encode(node.getId(), StandardCharsets.UTF_8.name()));
-                style.append(".").append(className)
-                        .append(" .open { visibility: ").append(node.isOpen() ? "visible;}" : "hidden;}");
-
-                style.append(".").append(className)
-                        .append(" .closed { visibility: ").append(node.isOpen() ? "hidden;}" : "visible;}");
-
-                return Optional.of(style.toString());
-            } catch (UnsupportedEncodingException e) {
-                throw new UncheckedIOException(e);
-            }
+        if (node.getType() == Node.NodeType.SWITCH) {
```

Comment:
Is it normal that everywhere you remove "avoidSVGComponentsDuplication" check?

Reply:
No, My mistake